docs/theme/static/static.js

```diff
@@ -567,9 +567,9 @@ steal(
         })
 
 
-        $('#bithub-events-embed').bithubEmbed({
-            hubId: 5,
-            tenant_name: 'charming_volcano_7196_1'
+        $('#bithub-events-embed').upcomingEvents({
+            apiKey: 'AIzaSyBsNpdGbkTsqn1BCSPQrjO9OaMySjK5Sns',
```

Comment:
Confirmed that the key works correctly when locked down to `donejs.com/community.html` as a referrer. 🎉